types/dispatchr/addons/BaseStore.d.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference types="node" />
+
+import { Dispatcher, DispatcherInterface, DispatcherContext, Store } from '../index';
+import { EventEmitter } from 'events';
+
+declare class BaseStore<S = {}> extends EventEmitter implements Store<S> {
+    constructor(dispatcher: DispatcherInterface);
+    initialize?: () => void;
```

Comment:
`initialize? (): void;`

If both the class and `initialize` were marked abstract, would the class be subclassable without needing to declare `initialize`?

`BaseStore` does provide all of those listed methods but allows extenders to provide an `initialize` method.